examples/functions/01_basic_oracle/src/actions/refresh_prices.rs

```diff
@@ -4,39 +4,37 @@ use crate::*;
 pub struct RefreshPrices<'info> {
     #[account(
         seeds = [PROGRAM_SEED],
-        bump = program.load()?.bump,
-        constraint = program.load()?.is_valid_enclave(&quote.load()?.mr_enclave) @ BasicOracleError::InvalidMrEnclave
+        bump = program_state.load()?.bump,
+        // has_one = function @ BasicOracleError::IncorrectSwitchboardFunction,
```

Comment:
you want this removed?

Reply:
because program_state is a PDA i would have to restart my localnet each time which takes ~2mins when using the local switchboard programs so i comment this out for now until its wrapped up